src/ontology/metadata/icd11.yml

```diff
@@ -0,0 +1,13 @@
+id: ICD11
+label: International Classification of Diseases 11th Revision
+prefix_map:
+  ICD11: http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/ICD11/
+description: >
```

Comment:
@sabrinatoro @twhetzel Here's the ICD11 description, as well as the label and homepage, that I decided on. Just following up per our call.

Some of these `metadata/*.yml` files say something about the data source in general, and other of them instead say a bit about the data artefact / its flavor / where it was downloaded. Anyway it's probably not super important to flesh this out, but this is what I went with: a description that popped up on Google, and a note about what flavor of ICD11 this is.